examples/angular-cli/src/stories/custom-metadata.stories.ts

```diff
@@ -64,3 +64,40 @@ storiesOf('Custom ngModule metadata', module)
       },
     };
   });
+
+storiesOf('Common ngModule metadata', module)
```

Comment:
Can you please extract it to a separate file? So it will play nicely with #2885 and #2918

Reply:
I agree, it'd  be better to have a separate example that shows:

1) An example with this decorator (basic usage)
2) Decorator + Knobs addon
3) Decorator + Knobs + `template`
4) Decorator + knobs + `component`

We don't have official docs for most of this and we usually point people to the examples so we try to keep as many as we can with all the use cases

Reply:
I guess I'm not really seeing what value these examples would have? All this decorator does is allow you to declare a set of metadata that is shared amongst your subsequent stories. Whether you're using knobs, template, component, etc. is completely orthogonal to it. Since there is already examples of those things, I'm not sure how 

Wouldn't it make more sense to modify all the other examples to use this decorator where appropriate, i.e. instead of supplying duplicate metadata, as in `custom-pipes` and `custom-providers`? Then the metadata example file could just be:

 1. An example with metadata supplied on the story.
 2. Metadata supplied using this decorator.
 3. Metadata supplied on the story that combines with that from the decorator.